docker-app/qfieldcloud/core/models.py

```diff
@@ -895,7 +897,10 @@ class Meta:
     @property
     def thumbnail_url(self):
         if self.thumbnail_uri:
-            return get_s3_object_url(self.thumbnail_uri)
+            return reverse_lazy(
```

Comment:
same

Reply:
There is a strong bias towards `reverse_lazy` in the codebase so far, so I keep it.